libmscore/fraction.h

```diff
@@ -233,7 +233,8 @@ class Fraction {
             // MScore::division     - ticks per quarter note
             // MScore::division * 4 - ticks per whole note
             // result: rounded (MScore::division * 4 * _numerator * 1.0 / _denominator) value
-            const auto result = (static_cast<int_least64_t>(_numerator) * MScore::division * 4 + (_denominator/2)) / _denominator;
+            const int sgn = (_numerator < 0) ? -1 : 1;
```

Comment:
Wouldn't it be better to put this ternary operator directly in the line below, rather than creating a new const for it? That we we don't have to touch RAM an extra time, AFAIK.

Reply:
ticks is called very often in renderMidi, and I'd imagine is still fairly frequent in other areas of the code. I'd say it isn't exactly critical, but there would be a noticeable difference in rendering playback for some medium size scores if it took 1ms longer.